src/data/web3.ts

```diff
@@ -14,6 +14,7 @@ const wcV2InitOptions: any = {
   version: 2,
   projectId: '72b7b3359ab477e339a070f615806aa6',
   requiredChains: [1, 56],
+  dappUrl: "https://near.org"
```

Comment:
Thanks @ciocan for your contribution! The only question I have is this works for the testnet?

Reply:
I mean we do have a https://test.near.org and I think this `dappUrl` won't work. But maybe we don't need it either 🧐
@mpeterdev WDYT?

Reply:
might as well make it a ternary driven by the current net

Reply:
Thanks @shelegdmitriy for pointing it out - I added the testnet ternary.
I also removed from config the version which was not used in the new docs and requiredChains which is problematic for MetaMask mobile because it rejecta transactions.